templates/apiary-hms-readonly.json

```diff
@@ -26,42 +26,82 @@
       }
     ],
     "environment":[
-     {
+      {
         "name": "MYSQL_DB_HOST",
         "value": "${mysql_db_host}"
-     },
-     {
+      },
+      {
         "name": "MYSQL_DB_NAME",
         "value": "${mysql_db_name}"
-     },
-     {
+      },
+      {
         "name": "MYSQL_SECRET_ARN",
         "value": "${mysql_secret_arn}"
-     },
-     {
+      },
+      {
```

Comment:
Are all these new variables optional?

Reply:
Yes.  All these vars were previously passed into the read-write HMS ECS template (as either blank/not-set when Ranger is not configured, or with appropriate values for Ranger).  This code is just also now passing them into the read-only ECS template so that read-only HMS can use Ranger for auditing (if Ranger is configured).